examples/book.rb

```diff
@@ -20,12 +7,11 @@ def initialize(title, author, pages)
     @pages = pages
   end
 
-  def isAlongBook
+  #check if a book is long or not
+  def longBook?
     if @pages >= 100
       return true
     end
     return false
```

Comment:
By default, ruby always returns the value of the last line in a function unless a return statement is explicitly used. For this reason, it is common to simply write the desired result as the final line of the function, for example

```ruby
def longBook?
  ... logic
  false
end
```